Form-Controls/index.html

```diff
@@ -12,15 +12,38 @@
       <h1>Product Pick</h1>
     </header>
     <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+      <form action = "results.html" method="GET">
+        <div>
+          <label for="name">Name </label>
+          <input type="text" name="name" id="name" placeholder="YourName" required>
+        </div>
+        <div>
+           <label for="email"> Email </label>
+          <input type="email" name="email" id="email" placeholder="YourEmail" required>
+        </div>
+        <div>
+          <label for="T-shirt colour">T-shirt colour </label>
+          <select name="T-shirt_colour" id="T-shirt_colour">
+            <option value="Green"> Green </option>
```

Comment:
When developing a form, consistency is important 🙂. You have 'Green' that starts with an uppercase. However, the other colours all start with lowercase. Can you consolidate this so they are uniform?